packages/component/src/Composer.js

```diff
@@ -126,6 +126,7 @@ const Composer = ({
   attachmentRenderer,
   cardActionMiddleware,
   children,
+  dir,
```

Comment:
I think we should massage the input in `<Composer>` to make other code simpler..

For example, if the user send `<Composer dir="xyz">`, we should send `"auto"` to the context.